apps/api/src/lib/db/migrations/0040_flashy_reavers.sql

```
@@ -0,0 +1,59 @@
+	"batch_id",
+	'legacy:' || "batch_id",
+	0,
+	false,
+	false,
+	false,
+	'failed',
+	min("selected_at")
+FROM "discord_challenge_map_history"
+GROUP BY "batch_id";--> statement-breakpoint
+CREATE UNIQUE INDEX "discord_challenge_batches_daily_key_unique" ON "discord_challenge_batches" USING btree ("daily_key");--> statement-breakpoint
+ALTER TABLE "discord_challenge_map_history" ADD CONSTRAINT "discord_challenge_map_history_batch_id_discord_challenge_batches_id_fk" FOREIGN KEY ("batch_id") REFERENCES "public"."discord_challenge_batches"("id") ON DELETE cascade ON UPDATE no action;--> statement-breakpoint
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)